File: packages/Server/Application/Accounts/AccountProfile.cs
namespace Solidarity.Application.Accounts;

public class AccountProfile : Model
{
	public int AccountId { get; set; }

	public Account? Account { get; set; } = null!;

	[MaxLength(50)]
	public string FirstName { get; set; } = null!;

	[MaxLength(50)]
	public string LastName { get; set; } = null!;

	public DateTime? BirthDate { get; set; }
}

File: packages/Client/sdk/models/AccountProfile.ts
import { Account, Model, model } from "sdk"

@model('AccountProfile')
export class AccountProfile extends Model {
	constructor(accountId?: number) { 
		super()
		this.accountId = accountId
	}

	accountId?: number
	account?: Account
	firstName?: string
	lastName?: string
	birthDate?: Date
}